docs/integrations/creating-nodes/plan/node-types.md

```diff
@@ -4,14 +4,22 @@ contentType: explanation
 
 # Node types: Trigger and Action
 
-There are two node types you can build for n8n: app nodes and trigger nodes.
+There are two node types you can build for n8n: trigger nodes and action nodes.
 
 Both types provide integrations with external services. 
 
 ## Trigger nodes
 
 Trigger nodes start a workflow and supply the initial data. A workflow can contain multiple trigger nodes but with each execution, only one of them will execute, depending on the triggering event.
 
+There are three types of trigger nodes in n8n: 
+
+| Type | Description | Example Nodes |
+| --- | --- | --- |
+| Webhook | Nodes for services that support webhooks. These nodes listen for events and trigger workflows in real time. | [Zendesk Trigger](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Zendesk){:target=_blank .external-link}, [Telegram Trigger](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Telegram){:target=_blank .external-link}, [Brevo Trigger](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Brevo){:target=_blank .external-link} |
```

Comment:
@freakwriter this is actually intentional. This page is for users building community nodes who might be looking for specific trigger node/trigger node type they can reference or base their node on. So I thought it would be more helpful to link the codebase, so they can copy the code from there.

Reply:
Ahh okay!